testing/asserts.ts

```diff
@@ -882,7 +882,7 @@ export async function assertSnapshot(context: Deno.TestContext, actual: unknown)
     snapshotFile = file ? JSON.parse(file) : {};
   }
   const snapshot = snapshotFile?.[name];
-  if (!context.update) {
+  if (!Deno.args.includes('--update')) {
```

Comment:
```suggestion
  if (!Deno.args.some(arg => arg === '--update' || arg === '-u')) {
```

Could do something like this to support `-u` and `--update`.

Reply:
I will handle this after https://github.com/denoland/deno/pull/14007 merged!

Reply:
BTW to pass `-u` option to `Deno.args`, you need to invoke test command like `deno test -- -u` because the flags before `--` are consumed by Deno CLI itself. I'm ok with that for the first pass implementation, but it might be non ideal solution, I guess

Reply:
Right. I also noticed that the flag for updating snapshot can affect other codes that wanted to test.. but I think that using `Deno.args` is better than making an option on Deno test cli.

Reply:
Imo this is the best solution for first pass. But if/when snapshot testing is a more established then we should look to implement this on the Deno CLI instead; as @hyp3rflow mentioned, this can affect test execution.